### PR TITLE
Change viewpoints serializers and tests

### PR DESCRIPTION
**Pull request by AL9000**

### CODE REVIEW

File: terracommon/tropp/serializers.py [hunk 1]
```diff
@@ -106,30 +106,34 @@ class Meta(CampaignSerializer.Meta):
...
 
 
 class ViewpointSerializerWithPicture(serializers.ModelSerializer):
-    picture = SimplePictureSerializer(required=False, write_only=True)
+    picture_ids = serializers.PrimaryKeyRelatedField(
+        source='pictures',
+        queryset=Picture.objects.all(),
+        required=False,
+        many=True,
+        write_only=True,
```
Comment (jrmi) [inline terracommon/tropp/serializers.py]:
```suggestion
        write_only=False,
```

Comment (jrmi) [reply terracommon/tropp/serializers.py]: We want it read and write, isn't it ?

File: terracommon/tropp/serializers.py [hunk 2]
```diff
@@ -106,30 +106,34 @@ class Meta(CampaignSerializer.Meta):
...
     owner = serializers.ReadOnlyField(source='owner.email')
+    file = VersatileImageFieldSerializer('tropp')
 
     class Meta:
         model = Picture
         fields = '__all__'
 
 
 class SimplePictureSerializer(PictureSerializer):
-    file = VersatileImageFieldSerializer('tropp')
```
Comment (jrmi) [inline terracommon/tropp/serializers.py]: We may keep this for frontend ?

Comment (Al-Ouest) [reply terracommon/tropp/serializers.py]: Yes, it's already in `PictureSerializer` :)

File: terracommon/tropp/serializers.py [hunk 3]
```diff
@@ -25,11 +25,11 @@ def get_attribute(self, instance):
 
 class SimpleViewpointSerializer(serializers.ModelSerializer):
     picture = SerializerMethodField()
-    geometry = GeometryField(source='point.geom', read_only=True)
+    point = GeometryField(source='point.geom')
```
Comment (nahuelange) [inline terracommon/tropp/serializers.py]: Why read_only is dropped here ?

File: terracommon/tropp/serializers.py [hunk 4]
```diff
@@ -106,30 +106,33 @@ class Meta(CampaignSerializer.Meta):
...
-    file = VersatileImageFieldSerializer('tropp')
-
     class Meta:
         model = Picture
         fields = ('id', 'date', 'file', 'owner', 'properties')
 
 
 class ViewpointSerializerWithPicture(serializers.ModelSerializer):
-    picture = SimplePictureSerializer(required=False, write_only=True)
+    picture_ids = serializers.PrimaryKeyRelatedField(
```
Comment (nahuelange) [inline terracommon/tropp/serializers.py]: Isn't it a related model ?

File: terracommon/tropp/serializers.py (hunk 3 above)
Comment (SebCorbin) [reply terracommon/tropp/serializers.py]: `SimpleViewpointSerializer` is only used in `DetailCampaignNestedSerializer` and on list action on viewpoints

File: terracommon/tropp/serializers.py (hunk 4 above)
Comment (SebCorbin) [reply terracommon/tropp/serializers.py]: Yes, it is, why this question?

File: terracommon/tropp/serializers.py (hunk 2 above)
Comment (jrmi) [reply terracommon/tropp/serializers.py]: Yes but the ViewpointSerializerWithPicture use the simple one

Comment (AL9000) [reply terracommon/tropp/serializers.py]: Yes, and the simple one is a subclass of PictureSerializer :
` class SimplePictureSerializer(PictureSerializer)`
So the definition of `file` should be correct.

Comment (jrmi) [reply terracommon/tropp/serializers.py]: Ok, that the information I didn't see !

**Pull request merged**